src/pages/ReportDetailsPage.js

```diff
@@ -132,11 +132,9 @@ class ReportDetailsPage extends Component {
                             style={styles.reportDetailsTitleContainer}
                         >
                             <Avatar
-                                isChatRoom={isChatRoom}
-                                isArchivedRoom={ReportUtils.isArchivedRoom(this.props.report)}
                                 containerStyles={[styles.singleAvatarLarge, styles.mb4]}
                                 imageStyles={[styles.singleAvatarLarge]}
-                                source={this.props.report.icons[0]}
+                                source={OptionsListUtils.getAvatarSources(this.props.report)}
```

Comment:
`OptionsListUtils.getAvatarSources` returns an array, right? But `Avatar`'s `source` prop should be a string or function. Something seems wrong here ... should it be like this?

```suggestion
                                source={_.first(OptionsListUtils.getAvatarSources(this.props.report))}
```

Reply:
Oh good catch, I think I missed this when I merged `getAvatarSources` functions. Not sure why I didn't see an error when I tested